typescript-conditionals/main.ts

```diff
@@ -63,7 +63,7 @@ function isOldEnoughToDrinkAndDrive(person: Person): boolean {
 function categorizeAcidity(pH: number): string {
   if (pH === 7) {
     return 'neutral';
-  } else if (pH <= 7) {
+  } else if (pH < 7) {
```

Comment:
This is still saying that all numbers below 7 are acid. You were right with the `<=`, but what is the first number that is considered acidic?